policies/testing.md

```diff
@@ -42,6 +42,50 @@ A test may be omitted where writing the test would result in disproportionately
 more effort than writing the code being tested. For example, difficult to
 reproduce error conditions.
 
+Triage Labels
+-------------
+
+Before approving a PR, the applicability of the testing policy to the PR must
+be assessed and an appropriate label applied to the PR. One of the following
+labels must be applied:
+
+- `tests: present`, to be added where suitable tests are included in the same PR.
+- `tests: needed`, where the testing policy requires tests but they are not
+  currently included in a PR.
+- `tests: exempted`, where tests are not included in a PR but are not required
+  by this policy (for example, because the PR concerns only documentation,
+  or because writing the test would result in disproportionately more effort;
+  see above).
+- `tests: deferred`, where the tests are required by the testing policy
+  but it is intended to add these tests in a subsequent PR.
+- `tests: added in subsequent PR`, where the tests were deferred to a subsequent
```

Comment:
We don't need this one.  If the tests have been merged, presumably the build is now broken (which is very bad) or the tests aren't testing what this PR is changing (which makes this tag inappropriate).

It would be very rare where we'd merge the tests before the PR that needs them.  In that case, changing the tag to "tests: exempt" and adding a note would be adequate.

As far as I can remember, the couple of times this might have been possible, we always leave the tests unmerged until the PR is.

Reply:
When reading the whole explanation (which talks about *subsequent* PRs, not a test PR that has been merged beforehand), this label makes sense to me.
As I understand it, this is a way to confirm that the test for an already merged PR that has the `test: deferred` label now has the corresponding test.

This makes it easy to search for PRs (merged or not) in lack of a test and write one; just search for `test: deferred`, write the test, make a PR for it, merge it, go back to the PR that was missing a test and change the `test: deferred` label to `test: added in subsequent PR`, and no one else will think there's a test to be written.

Reply:
Why not simply remove the test: deferred label in that case and save label pollution?

Reply:
Ok, gotta agree that simply removing `test: deferred` would work too.  It's also possible to add a comment saying something like "test added by PR #nnn"